docs/docs/References/Classes.md

```diff
@@ -154,9 +154,9 @@ from dlpx.virtualization.platform import Plugin
 plugin = Plugin()
 
 @plugin.linked.pre_snapshot()
-def linked_pre_snapshot(staged_source, repository, source_config, snapshot_parameters):
-  if snapshot_parameter.resync:
-      print(snapshot_parameter.resync)
+def linked_pre_snapshot(staged_source, repository, source_config, optional_snapshot_parameters):
+  if optional_snapshot_parameters is not None and optional_snapshot_parameters.resync:
+    print(optional_snapshot_parameters.resync)
```

Comment:
Could you check the description here, it seems to imply that `resync` is required and that `resync` is set to false by policy. I think the `optional_snapshot_parameters` are set to `None` by policy?

> ## SnapshotParametersDefinition
> User provided parameters for the snapshot operation. It includes a boolean property named `resync` that can be used to indicate to the plugin whether or not to initiate a full ingestion of the dSource. The parameters are only set during a manual snapshot. When using a sync policy, `resync` defaults to `false`.
>

Reply:
Discussed offline. Removing this section on SnapshotParametersDefinition since it's stale and defined in Schemas_and_Autogenerated_Classes.md.